docs/migration-guides/qiskit-backend-primitives.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Migrate to primitives for third-party provider users
+description: Migrate to primitives for users of third-party providers with `backend.run`
+
+---
+
+# Migrate to primitives for users of third-party providers with `backend.run`
+
+The `backend.run()` interface, previously the primary access point for IBM&reg; QPUs, has been superseded 
+by the Qiskit [primitives interface](/guides/primitives) in `qiskit_ibm_runtime`. 
+While this has become the standard for IBM providers, not all third-party providers have transitioned 
+to the primitives framework, and `backend.run()` remains in Qiskit for backward compatibility.
+
+This guide is for you if you have written or want to write code using `Sampler` and `Estimator` 
+primitives but need to interact with a provider that still relies on `backend.run()`. If you are a 
+provider developer looking to implement and expose your own `Sampler` and 
+`Estimator` primitives, you can find guidance in the [Migrate provider interfaces from backend.run to primitives](/migration-guides/external-providers-primitives-v2) guide.  
+
+For providers that don't expose native primitives, the Qiskit SDK offers wrappers for `backend.run` that you can 
+use out-of the box: the [`BackendSamplerV2`.](/api/qiskit/qiskit.primitives.BackendSamplerV2) and 
+[`BackendEstimatorV2`](/api/qiskit/qiskit.primitives.BackendEstimatorV2) classes.
+
+If you are already familiar with the [`SamplerV2`](/api/qiskit-ibm-runtime/sampler-v2) 
+and [`EstimatorV2`](/api/qiskit-ibm-runtime/estimator-v2) interfaces, the transition to 
+[`BackendSamplerV2`.](/api/qiskit/qiskit.primitives.BackendSamplerV2) and 
+[`BackendEstimatorV2`](/api/qiskit/qiskit.primitives.BackendEstimatorV2) is seamless, as
+these classes follow the same usage patterns. The inputs to the primitives should follow 
+the Primitive Unified Bloc (PUB) syntax, and the output formats should follow the
+V2 primitives standard. See the [Overview of PUBs section](/guides/primitive-input-output#pubs) in 
+the Primitive inputs and outputs guide for details.
+
+The two main differences with the implementations of [`SamplerV2`](/api/qiskit-ibm-runtime/sampler-v2) 
+and [`EstimatorV2`](/api/qiskit-ibm-runtime/estimator-v2) are:
+
+1.  The `backend` input passed during the "backend primitive" instantiation. This input argument allows to 
+internally wrap the access to the internal provider through `backend.run()` and it should not be confused
+with the `mode` input argument of [`SamplerV2`](/api/qiskit-ibm-runtime/sampler-v2) 
+and [`EstimatorV2`](/api/qiskit-ibm-runtime/estimator-v2).
+
+2. The use of `options`, which contains one more level of nesting compared to provider-specific primitives and
```

Comment:
I don't understand where this additional level of nesting comes from, given you can also do `options={"default_shots": 1024}` using qiskit-ibm-runtime.

Reply:
Additional level of nesting is probably not the right term, I wanted to find a nice way to explain that the structure of the options is sometimes different. The level of nesting point referred to `run_options` which may bundle options that are "first level" in the runtime primitives. But I might be confusing things. It might be better to simply lay out how the options look like. The paragraph below does explain that custom option setting might require a subclass anyways.